### ISSUE TITLE
testing 3492: change UX of entering questions: check to agree to terms of use not accessible

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#3285]
**URL:** https://meinberlin-dev.liqd.net/projekte/module/interaktive-veranstaltung-11/
**user:** all, using software without mouse
**expected behaviour:** I can check/uncheck my agreement to terms of use without using my mouse
**behaviour:** if focus is on check and I hit enter, the questions is either sent (if check already marked) or I get a browser warning. I cannot change status of check without mouse.
**important screensize:**
**device & browser:** 
**Comment/Question:** we need to be accessible on meinBerlin.

Screenshot?

**Comment by phillimorland** [comment]
@CarolingerSeilchenspringer keyboard command for check boxes is spacebar not enter

**Comment by CarolingerSeilchenspringer** [comment]
@phillimorland good to know, thanks!
